src/wms/src/main/java/org/geoserver/wms/capabilities/Capabilities_1_3_0_Transformer.java

```diff
@@ -682,10 +682,14 @@ private void handleLayers() {
             }
             handleRootCrsList(srs);
 
-            CloseableIterator<LayerInfo> layers;
-            layers = catalog.list(LayerInfo.class, filter);
+            // create layer groups filter
+            final Filter lgFilter = Predicates.acceptAll();
```

Comment:
I'm a little lost here, basically you mean using the same filter that is use for layers to filter the namespace ? This will also change the current behavior of which layer groups are shown in the capabilities is that something we want ?